docs/getting-started/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Getting started with Apache Gravitino"
+slug: /getting-started
+license: "This software is licensed under the Apache License version 2."
+---
+
+There are several options for getting started with Apache Gravitino.
+
+<!--Docker option-->
+Installing and configuring Hive and Trino can be a little complex.
+If you are unfamiliar with the technologies, using Docker might be a good choice.
+There are pre-packaged containers for Gravitino, Apache Hive, Apache Hadoop,
+Trino, MySQL, PostgesSQL, and others,
+check [installing Gravitino playground](./playground.md) for more details.
```

Comment:
Breaking sentance over multipel lines and missing a space after the comma may cause issues.

Reply:
I would change it to " and others. Check"

Reply:
Fine. Will do.